modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase

```diff
@@ -384,6 +669,101 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12218. Admin can view channel currency matches catalogue currency but dosen't change the price list on a product's SKU details page"
```

Comment:
Admin can view channel currency matches catalog currency but **doesn't** change the price list on a product's SKU details page